### ISSUE TITLE
Post editing: side-by-side view glitch

### ISSUE THREAD

**Issue by mkalam-alami** [alakajam-team/alakajam#444]
Must be CSS-related (z-index shenanigans or stuff)

![firefox_QtjNklwaB0](https://user-images.githubusercontent.com/694508/64486716-da0c1980-d230-11e9-8670-089bc2a3bf66.png)

**Comment by mkalam-alami** [comment]
Fix:

```css
.editor-preview-active-side {
    background-color: #F5F5F5;
    padding: 10px;
}
```

**Comment by mkalam-alami** [comment]
Fixed with wrongly tagged 8b68b6d9
